art/readme.txt

```diff
@@ -16,11 +16,11 @@
                  /          _/           /             
 
 HEY
-Thanks for downloadin Friday Night FUNKIN'
-If you downloaded this from any other place than ninja-muffin24.itch.io/FUNKIN
+Thanks for downloadin Friday Night FUNKIN':Psych Engine
+If you downloaded this from any other place than https://gamebanana.com/mods/309789
```

Comment:
and also, psych engine is not only available for gamebanana
my bad, this PR is definitely useless just by modifying a file that nobody reads 💀